app/src/main/java/com/verygoodsecurity/demoapp/StartActivity.kt

```diff
@@ -44,6 +45,12 @@ class StartActivity : AppCompatActivity(R.layout.activity_start) {
         binding.llCollectViewPagerFlow.setOnClickListener {
             startActivity(VGSViewPagerActivity::class.java)
         }
+        binding.llDateRangeViewPagerFlow.setOnClickListener {
+            startActivity(DateRangeActivity::class.java)
+        }
+        binding.llCollectActivityFlow.setOnClickListener {
```

Comment:
@DonaldRG Duplication. Same in line [39](https://github.com/verygoodsecurity/vgs-collect-android/pull/524/files#diff-b34258b43f10238f0741356d81bce8fc8a4e1f43c86209c3be68076750a0084dR39)